README.md

```diff
@@ -1 +1,142 @@
+|Should| Use a forward slash in paths. | This is to ensure comptability on both Windows and Linux.
+|Should Not| Use aliases. |
+
+# Code Layout and Formatting
+This section provides an overview of the following:
+
+| Section Header | Description |
+| - | - |
+| EditorConfig | EditorConfig is used to enforce a consistant readable format. |
+| VS Code Settings | |
+| Comment Based Help | |
+| PlatyPS | Provides details of how to create comment based help and generate external help markdown using PlatyPS. |
+| Naming Conventions | Provides a table containing the case type to use for identifiers as well as examples of each. |
+
+## EditorConfig
+In order to maintain a consistent coding style an EditorConfig file is used to define the coding style to be used. The file **.editorconfig** contains the required styles. The EditorConfig file defines styles such as indentation size, indentation style, newline rules and more.
+
+### EditorConfig Installation
+
+EditorConfig can be installed as a VS Code extension. Search for and install **EditorConfig for VS Code**. The VS Code Marketplace has more information [here](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig).
+
+For further information on EditorConfig [EditorConfig](https://editorconfig.org/)
+
+## VS Code Settings
+
+TO DO
+
+## Comment Based Help
+Scripts and functions should contain comment based help that is compatible with **Get-Help**.
+
+The help should consist of the following elements:
+
+- Synopsis
+- Description
+- A parameter description for each parameter in the Param() block
+- At least one example demonstrating how the script can be executed
+
+For further information see [about_Comment_Based_Help](https://github.com/PowerShell/PowerShell-Docs/blob/staging/reference/5.1/Microsoft.PowerShell.Core/About/about_Comment_Based_Help.md)
+
+INSERT EXAMPLE
+
+## PlatyPS
+
+All PowerShell scripts in the Infrastructure-Scripts folder must have external help documentation generated in markdown using the [PlatyPS](https://github.com/PowerShell/platyPS) module.
+
+TO DO
+
+ADD A PESTER TEST FOR THIS TO CHECK DOC EXISTS
+
+## Naming Conventions
+To ensure a consistant readable format, use the following naming conventions:
+
+| Identifier                     | Case      | Example      |
+| ------------------------------ | --------- | ------------ |
+| Module Names                   | Pascal    |              |
+| Function or cmdlet names       | Pascal    |              |
+| Class                          | Pascal    |              |
+| Attribute Names                | Pascal    |              |
+| Public fields or properties    | Pascal    |              |
+| Global variables               | Pascal    |              |
+| Constants                      | Pascal    |              |
+| Parameter variable             | Pascal    |              |
+| Variables                      | camel    |              |
```

Comment:
Can this be Local Variables to differentiate from Global Variables?

Reply:
I've updated the README.md on my remote branch. Improving the naming convention table and will commit.